pkgs/applications/networking/flent/default.nix

```diff
@@ -10,13 +39,25 @@ buildPythonApplication rec {
     sha256 = "1llcdakk0nk9xlpjjz7mv4a80yq4sjnbqhaqvyj9m6lbcxgssh2r";
   };
 
-  buildInputs = [ netperf ];
+  outputs = [ "out" ] ++ stdenv.lib.optional withMan "man";
+
+  buildInputs = [ makeWrapper ] ++ stdenv.lib.optionals withMan [ sphinx ];
+
   propagatedBuildInputs = [
     matplotlib
     procps
     pyqt5
   ];
 
+  postBuild = stdenv.lib.optionalString withMan ''
+    make man
```

Comment:
This `postBuild` is not necessary and can be removed. The manual is built automatically.